[2024-06-21T15:35:20.453+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-21T15:30:12.868882+00:00 [queued]>
[2024-06-21T15:35:20.459+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-21T15:30:12.868882+00:00 [queued]>
[2024-06-21T15:35:20.460+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-06-21T15:35:20.481+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2024-06-21 15:30:12.868882+00:00
[2024-06-21T15:35:20.487+0000] {standard_task_runner.py:57} INFO - Started process 23228 to run task
[2024-06-21T15:35:20.493+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'scheduled__2024-06-21T15:30:12.868882+00:00', '--job-id', '271', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmpp0nghmjj']
[2024-06-21T15:35:20.495+0000] {standard_task_runner.py:85} INFO - Job 271: Subtask kafka_data_stream
[2024-06-21T15:35:20.520+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 271 for task kafka_data_stream (DagRun for kafka_spark_dag with run_id or execution_date of 'scheduled__2024-06-21T15:30:12.868882+00:00' not found; 23228)
[2024-06-21T15:35:20.526+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-06-21T15:35:20.542+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
